### PR TITLE
Decrease spacing between the timestamp table header and the sorting icon

### LINKED ISSUE #12221: Decrease spacing between the timestamp table header and the sorting icon
### Problem to solve

Following up from https://github.com/gitpod-io/gitpod/pull/12135#discussion_r949299959, we need to decrease spacing between the timestamp table header and the sorting icon.

| BEFORE | AFTER |
|-|-|
| header-before | header-after |

### PR DESCRIPTION
**Pull request by ashwinair**
## Description

re-opening - #12244
## Related Issue(s)

Fixes #12221

## Release Notes

```release-note
Decrease spacing between the timestamp table header and the sorting icon
```

## Werft options:

- [x] /werft with-preview

### CODE REVIEW

**Comment by gtsiolis** [comment]
Thanks for taking the time to reopen this, @ashwinair! 

Let me spin up an environment for this.

/werft run with-preview with-payment

**Comment by gtsiolis** [comment]
Also, re-posting from https://github.com/gitpod-io/gitpod/pull/12244#issuecomment-1222082130:

> You'll also need to sign a Contributor License Agreement (CLA)[[1](https://en.wikipedia.org/wiki/Contributor_License_Agreement)] once before merging your first contribution. Looping in @meysholdt to reach out about the CLA. 🏓

**Comment by ashwinair** [comment]
@gtsiolis, can you please review 🧡 

also a quick (stupid) question:
can I use special characters like - or /  for naming my branch while contributing

**Comment by gtsiolis** [comment]
> can I use special characters like - or / for naming my branch while contributing

@ashwinair Yes, builds will complete successfully when using `/` in branch names! ✔️

**Review by gtsiolis — APPROVED** [review]
UX LGTM, @ashwinair! ✨ 

Approving to unblock merging but this needs to more actions:
1. `@meysholdt` reaching out for the CLA signing process before merging this first contribution.
2. Someone from `@gitpod-io/engineering-webapp` to additionally approve code changes. Approving code changes is already covered in https://github.com/gitpod-io/gitpod/pull/12244#pullrequestreview-1079926706 but needed for merging this PR. Thanks @geropl!

| BEFORE | AFTER |
|-|-|
| Screenshot 2022-08-22 at 3 11 06 PM (2) | Screenshot 2022-08-22 at 3 11 38 PM (2) |

/hold

**Review by geropl — APPROVED** [review]
LGTM (again) :upside_down_face:

**Comment by gtsiolis** [comment]
Closing this as it's no longer relevant because we've removed the sorting icon. Thanks again, @ashwinair!

**Pull request closed without merging**
